taskipy/exceptions.py

```diff
@@ -30,6 +34,17 @@ def __init__(self, task_name: str):
     def __str__(self):
         return f'could not find task "{self.task}"'
 
+class InvalidUsageError(TaskipyError):
+    exit_code = 127
+
+    def __init__(self, parser: ArgumentParser):
+        super().__init__()
+        self.__parser = parser
+
+    def __str__(self):
+        with StringIO() as buf, redirect_stdout(buf):
+            self.__parser.print_usage()
+            return buf.getvalue().strip('\n')
```

Comment:
This is a clever trick! Definitely gonna remember that if I ever need it.

That said, it can be made easier using the [format_usage](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.format_usage) method, which returns the usage string (instead of printing it).

```suggestion
        return self.__parser.format_usage().strip('\n');
```

Reply:
Good catch! That's much clearer. I'll try to get that updated sometime tonight